lib/Retcon/Diff.hs

```diff
@@ -117,20 +120,27 @@ ignoreConflicts :: MergePolicy
 ignoreConflicts = MergePolicy {..}
   where
     extractLabel = const Unamed
-    merge p1 p2
-      = let m1 = toMap p1
-            m2 = toMap p2
-            allOps    = M.unionWith (++) m1 m2
-            conflicts = M.intersectionWith (++) m1 m2
-            accepts   = M.difference allOps conflicts
-        in  (fromMap accepts, justOps conflicts)
+    mergeWith :: [Patch] -> MergeResult
+    mergeWith ps =
+        let ops    = map toMap ps
+            allOps = mapReduce (M.unionWith (++)) ops
+            ignore = M.filter (\l -> 1 < length l) allOps
```

Comment:
I don't think this works. If `patch1` has `delete someval 1, insert someval 0` and `patch2`, `patch3` etc. don't mention `someval` at all, this is gonna ignore the `someval` changes